tools/Ecoregionalization_workflow/.shed.yml

```diff
@@ -0,0 +1,64 @@
+categories:
+    - Ecology
+description: Tools to compute ecoregionalization
+long_description: |
+  Tools to compute ecoregionalization with BRT model predictions and clustering.
+name: ecoregionalization workflow 
+owner: ecology 
+remote_repository_url: https://github.com/PaulineSGN/Workflow_Galaxy/tree/main
+type: unrestricted
+
+repositories:
```

Comment:
please remove those repository entries and convert this file into a suite-based one ... see for example here https://github.com/galaxyproject/tools-iuc/blob/main/tools/anndata/.shed.yml#L12